bin/format_coverage.dart

```diff
@@ -66,6 +68,7 @@ Future<void> main(List<String> arguments) async {
   final hitmap = await HitMap.parseFiles(
     files,
     checkIgnoredLines: env.checkIgnore,
+    ignoreGlobs: env.ignoreFiles,
```

Comment:
Why not add this logic in the same place as `reportOn`? You could just add it to the `_getPathFilter` function in formatter.dart. Having the logic in two totally different places makes it a bit hard to read.